docs/plugin-proposal-dynamic-import.md

```diff
@@ -0,0 +1,103 @@
+---
+id: babel-plugin-proposal-dynamic-import
+title: "@babel/plugin-proposal-dynamic-import"
+sidebar_label: dynamic-import
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> **NOTE**: This plugin is included in `@babel/preset-env`, in [ES2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md).
+
+Transforms `import()` expressions to non-ESM module formats.
+
+## When (not) to use this plugin
+
+If you are using Babel in production, you should normally not use this plugin. Instead, you should use
+a bundler, such as Webpack, Rollup or Parcel to handle `import()` expressions. However, there are still some valid use cases:
```

Comment:
```suggestion
If you are using a bundler, such as Webpack, Rollup or Parcel, you should **not** use this plugin and let your bundler handle `import()` expressions.

You should use this plugin if:
```

all the listed use cases can be in production, and many people use bundlers in development :stuck_out_tongue: